package.json

```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "start": "node src/cli/index.js",
     "test": "jest ./src",
+    "test:e2e:installs": "jest './scripts/e2e-installs.test.js'",
```

Comment:
Is there a reason to use single quotes rather than nothing?

Reply:
I can remove them, it works without quotes now. I think it used to get processed as a regex before but not anymore.